### DISCUSSION TITLE
Registry Remapping Failed

### DISCUSSION THREAD

**Discussion by Kingtoast1st** [FabricMC/fabric-api#2708] [Player support] [answered]
so I started a fabric Minecraft server for some friends while doing trial and error to find any mods that are corrupted, While doing this i started to get a error that says.
Registry remapping failed: Received ID map for minecraft:entity_type contains IDs unknow to the receiver! - minecraft:wolf

I have no idea whats is happening here and I have no idea how to Identify it and looking at the .bat console doesn't tell me anything or give me logs.

Server Mods file:
3dskinlayers
Advancementplaques
animal_feeding_trough
architectury
betterdroppeditems
BetterF3
BiomeParticleWeather
ClothConfig
Configured
Cosmetica
Coupling
DynamicSourroundingsRemastered
fabric API
ForgeConfigAPIPort
freecam
GuardVillagers
Iceberg
Journeymap
litematica
malilib
MapFrontiers
Merchantmarkers
Midnightlib
Modmenu
moonlight
morevillagers
MyVillagePack
optifabric
OptiFine
supplementaries

**Comment by warjort** [comment, marked as answer]
Maybe this mod is installed on your client?
https://github.com/MrKacafirekCZ/doggomod/issues/5

**Reply by Kingtoast1st** [reply]
Yes
